### DISCUSSION TITLE
现版本怎么下载原图，或怎么搞gp？

### DISCUSSION THREAD

**Discussion by Starseeker-H** [xiaojieonly/Ehviewer_CN_SXJ#1455] [Q&A]
右下角限额始终是0，开下载原图的情况下看都看不了，官网看gp只有250。My Home说下载高清晰度有ip限制（不知道理解的对不对）（这玩意之前有吗？没印象了），解除限制的方法……嗯，没看懂（
直接在官网图片下点下载原图提示下载需要gp，gp不足
![image](https://github.com/user-attachments/assets/676e01df-1133-405f-9384-defb3f3385c8)

（看的不明不白的，我这到底是被限额了还是没被限额啊？）

**Comment by HNPA2018** [comment]
同問，之前還是5000的上限，有一天突然就變成0了

**Comment by Pony-CW** [comment]
https://github.com/FooIbar/EhViewer/issues/1563#issuecomment-2310031101
